lblod/ember-rdfa-editor-lblod-plugins
# Snippet placeholder redesign

### Overview
Continued a little of the refactoring from replacing the snippet placeholder, so that the snippet placeholder component is now used inside a snippet component. This means that I could re-use the insert logic from the snippet buttons and if we wanted in the future we could include buttons in the same style as for inserted snippets. Since I was not including buttons for now, I was able to use @elpoelma 's redesign implementation work almost untouched.

For consistency, I move the snippet list ids information to RDFa for inserted snippet nodes.

##### connected issues and PRs:
Builds on https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/495
Jira ticket: https://binnenland.atlassian.net/browse/GN-5117

### Setup
N/A

### How to test/reproduce
Insert, replace, delete snippets and snippet placeholders with different configurations and make sure everything works as expected. The expected interactivity is different depending on which sample page:
- Besluit: It's only possible to insert snippets using the in-node insert button. placeholders cannot be inserted
- Reg statement: The in-node insert snippet button is hidden, but the sidebar button remains.
- Snippet edit: The in-node snippet insert button is shown, the sidebar button is not there. Placeholders can be inserted.

### Challenges/uncertainties
N/A

### Checks PR readiness
